test/edu-benefits/1990e/config/sponsorVeteran.unit.spec.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import ReactTestUtils from 'react-addons-test-utils';
+
+import { DefinitionTester, submitForm } from '../../../util/schemaform-utils.jsx';
+import formConfig from '../../../../src/js/edu-benefits/1990e/config/form';
+
+describe('Edu 1990e sponsorVeteran', () => {
+  const { schema, uiSchema } = formConfig.chapters.sponsorVeteran.pages.sponsorVeteran;
+  const definitions = formConfig.defaultDefinitions;
+  it('should render', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          data={{}}
+          uiSchema={uiSchema}
+          definitions={definitions}/>
+    );
+
+    expect(ReactTestUtils.scryRenderedDOMComponentsWithTag(form, 'input'))
```

Comment:
I need to update the applicant-info test to actually test for the right number of input fields. Was discusisng this with @jbalboni in PR #5036 and we agreed to be more specific. Take a look at Chris's test in benefitSelection.unit.spec.jsx